src/core/libmaven/PeakDetector.cpp

```diff
@@ -253,24 +253,43 @@ vector<mzSlice*> PeakDetector::processCompounds(vector<Compound*> set,
         return slices;
 }
 
-void PeakDetector::alignSamples() {
-        //being called in CLI.
-        if (mavenParameters->samples.size() > 1
-            && mavenParameters->alignSamplesFlag) {
-                cerr << "Aligning samples" << endl;
-
-                mavenParameters->writeCSVFlag = false;
-                processMassSlices();
-
-                cerr << "Aligner=" << mavenParameters->allgroups.size() << endl;
-                vector<PeakGroup*> agroups(mavenParameters->allgroups.size());
-                for (unsigned int i = 0; i < mavenParameters->allgroups.size(); i++)
-                        agroups[i] = &mavenParameters->allgroups[i];
-                //init aligner
-                Aligner aligner;
-                aligner.doAlignment(agroups);
-                mavenParameters->writeCSVFlag = true;
+void PeakDetector::alignSamples(const int& method) {
+
+    // only called from CLI
+    if(mavenParameters->samples.size() > 1 ) {
+
+        switch(method) {
+
+        case 1: {
+            ObiParams params("cor", false, 2.0, 1.0, 0.20, 3.40, 0.0, 20.0, false, 0.60);
```

Comment:
@rish9511 add a print statement for case 1 and 2. Important to let the user know what's happening in CLI. Especially because Obi-warp takes time.

Reply:
@shubhra-agrawal 
There are  print statements in the alignment algos somewhere. 
In case of obi-warp cli shows `Aligning Sample Retention times...` in the terminal
Do you think there is still a need for print statements ?

Reply:
@rish9511 Just make sure those are not debugging statements. User should know alignment is happening, that's all.